<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>600</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(88, 88, 88);
border-color: rgb(0, 0, 0);
selection-background-color: rgb(127, 172, 177);
border-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="main_grid">
   <widget class="QComboBox" name="targets">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="targets_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Targets</string>
    </property>
   </widget>
   <widget class="QLabel" name="xinit_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Initial Position</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="xinit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QLabel" name="plots_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>390</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Plots</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="model_group">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>351</width>
      <height>341</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Model Parameters</string>
    </property>
    <widget class="QLabel" name="nb_agent_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Number Of Agents</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setSysButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Set System</string>
     </property>
    </widget>
    <widget class="QLabel" name="current_model_label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Current Model</string>
     </property>
     <property name="buddy">
      <cstring>currentModel</cstring>
     </property>
    </widget>
    <widget class="QTextBrowser" name="currentModel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic; color:#9e9e9e;&quot;&gt;No system loaded&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="nbAgents">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="optimizer_group">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>371</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Optimizer Parameters</string>
    </property>
    <widget class="QPlainTextEdit" name="xmax">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>230</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="Horizon">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
    <widget class="QLabel" name="xmin_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Xmin</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="xmin">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>200</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="umax">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
    <widget class="QLabel" name="constraints_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Constraints</string>
     </property>
    </widget>
    <widget class="QLabel" name="umax_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Umax</string>
     </property>
    </widget>
    <widget class="QLabel" name="horizon_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Prediction Horizon</string>
     </property>
    </widget>
    <widget class="QLabel" name="xmax_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Xmax</string>
     </property>
    </widget>
    <widget class="QLabel" name="umin_label">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Umin</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="umin">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="obstacles">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="obstacles_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Obstacles</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delObsButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>290</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressources/icons.qrc">
       <normaloff>:/icons/del.svg</normaloff>:/icons/del.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="addObsButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>290</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="ressources/icons.qrc">
       <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="nsim_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Simulation Length</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="Nsim">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
    <widget class="QLabel" name="R_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="P">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>104</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
    <widget class="QLabel" name="P_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>P</string>
     </property>
    </widget>
    <widget class="QLabel" name="constraints_label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tuning Matrices</string>
     </property>
    </widget>
    <widget class="QLabel" name="Q_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Q</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="Q">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>104</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="R">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>104</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>16</width>
       <height>121</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="plots">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>430</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="launchButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>480</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>480</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
    <property name="icon">
     <iconset theme="add">
      <normaloff>.</normaloff>.</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>480</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>480</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(121, 173, 173);
background-color: rgb(100, 100, 100);
gridline-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="addTargetButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressources/icons.qrc">
      <normaloff>:/icons/add.svg</normaloff>:/icons/add.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="delTargetButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>390</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressources/icons.qrc">
      <normaloff>:/icons/del.svg</normaloff>:/icons/del.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <zorder>optimizer_group</zorder>
   <zorder>model_group</zorder>
   <zorder>targets</zorder>
   <zorder>targets_label</zorder>
   <zorder>xinit_label</zorder>
   <zorder>xinit</zorder>
   <zorder>plots_label</zorder>
   <zorder>plots</zorder>
   <zorder>launchButton</zorder>
   <zorder>stopButton</zorder>
   <zorder>pauseButton</zorder>
   <zorder>progressBar</zorder>
   <zorder>addTargetButton</zorder>
   <zorder>delTargetButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgb(127, 172, 177);</string>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionSetSimulationStructure"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_simulation_data">
     <property name="title">
      <string>&amp;save</string>
     </property>
     <addaction name="actionSaveOptimizerStructure"/>
     <addaction name="actionSaveSimulationData"/>
     <addaction name="actionSaveAnimation"/>
    </widget>
    <widget class="QMenu" name="menuLoad">
     <property name="title">
      <string>load</string>
     </property>
     <addaction name="actionLoadDefault"/>
     <addaction name="actionLoadMpc"/>
    </widget>
    <addaction name="menu_simulation_data"/>
    <addaction name="menuLoad"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSetSimulationStructure">
   <property name="text">
    <string>&amp;Simulation Structure</string>
   </property>
  </action>
  <action name="action_figures">
   <property name="text">
    <string>&amp;figures</string>
   </property>
  </action>
  <action name="actionSaveOptimizerStructure">
   <property name="text">
    <string>&amp;optimizer structure</string>
   </property>
  </action>
  <action name="actionSaveSimulationData">
   <property name="text">
    <string>&amp;simulation data</string>
   </property>
  </action>
  <action name="actionSaveOptimizerStructure_2">
   <property name="text">
    <string>optimizer structure</string>
   </property>
  </action>
  <action name="actionSaveAnimation">
   <property name="text">
    <string>animation</string>
   </property>
  </action>
  <action name="actionLoadDefault">
   <property name="text">
    <string>example</string>
   </property>
  </action>
  <action name="actionLoadMpc">
   <property name="text">
    <string>mpc</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
